tests/TraceContext/W3CTestService/trace-context-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -x
+
+function install_symfony() {
+    curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.alpine.sh' | bash >/dev/null
```

Comment:
I'm not familiar with `dl.cloudsmith.io` - downloading from a 3rd party script source feels dangerous to me.  Could we potentially have this script be in our repo?

Reply:
Looks like it's officially supported by Symfony, I think they're using the platform for artifact management beyond GitHub: https://symfony.com/download ; https://cloudsmith.com/

Reply:
Good enough for me